packages/storybook/src/nimble/table/table-matrix.stories.ts

```diff
@@ -158,6 +160,21 @@ export const tableMultipleSelectionThemeMatrix: StoryFn = createMatrixThemeStory
 );
 tableMultipleSelectionThemeMatrix.play = playFunction;
 
+const tableKeyboardFocusStates = cartesianProduct([
+    ['multiple'],
```

Comment:
not sure why these aren't referencing the enum but can you use the enum ref? `TableRowSelectionMode.multiple`
would be nice to do a quick clean-up of the other references in the file if possible.

Reply:
Updated